### PR TITLE
🌍 : Update Russian Translation

### PR DESCRIPTION
**Pull request by virtuman**
# Pull Request Template

## Summary

Add missing translations in Ru.tsx

## Change Type

Please delete any irrelevant options.

- [x] Bug fix (non-breaking change which fixes an issue)

## Testing

tested by viewing the pages on desktop and mobile

## Checklist

Please delete any irrelevant options.

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented in any complex areas of my code
- [x] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [x] I have written tests demonstrating that my changes are effective or that my feature works
- [x] Local unit tests pass with my changes
- [x] Any changes dependent on mine have been merged and published in downstream modules.
- [x] New documents have been locally validated with mkdocs

### CODE REVIEW

**Comment by danny-avila** [comment]
`com_nav_my_files` already exists for "My Files."

Please use this instead and remove the additional field for the same translation, and use `com_nav_my_files` instead, then I will merge.

File: client/src/components/Chat/Input/Files/FilesView.tsx [hunk 1]
```diff
@@ -27,7 +30,7 @@ export default function Files({ open, onOpenChange }) {
       <DialogContent className={cn('overflow-x-auto shadow-2xl dark:bg-gray-900 dark:text-white')}>
         <DialogHeader>
           <DialogTitle className="text-lg font-medium leading-6 text-gray-900 dark:text-gray-200">
-            My Files
+            {localize('com_ui_my_files')}
```
Comment (danny-avila) [inline client/src/components/Chat/Input/Files/FilesView.tsx]: use `com_nav_my_files`

File: client/src/components/Chat/Input/Files/FilesView.tsx [hunk 2]
```diff
@@ -4,8 +4,11 @@ import { Dialog, DialogContent, DialogHeader, DialogTitle } from '~/components/u
 import { useGetFiles } from '~/data-provider';
 import { DataTable, columns } from './Table';
 import { cn } from '~/utils/';
+import { useLocalize } from '~/hooks';
```
Comment (danny-avila) [inline client/src/components/Chat/Input/Files/FilesView.tsx:6]: please move useLocalize import before utils import

File: client/src/localization/languages/Eng.tsx [hunk 3]
```diff
@@ -33,6 +33,7 @@ export default {
     'Cannot attach file. Create or select a conversation, or try refreshing the page.',
   com_ui_examples: 'Examples',
   com_ui_new_chat: 'New chat',
+  com_ui_my_files: 'My Files',
```
Comment (danny-avila) [inline client/src/localization/languages/Eng.tsx]: remove

File: client/src/localization/languages/Ru.tsx [hunk 4]
```diff
@@ -28,6 +28,7 @@ export default {
   com_ui_experimental: 'Экспериментальный',
   com_ui_input: 'Ввод',
   com_ui_close: 'Закрыть',
+  com_ui_my_files: 'Мои файлы',
```
Comment (danny-avila) [inline client/src/localization/languages/Ru.tsx]: remove

**Review by danny-avila — CHANGES_REQUESTED** [review]

File: client/src/components/Chat/Input/Files/FilesView.tsx (hunk 2 above)
Comment (virtuman) [reply client/src/components/Chat/Input/Files/FilesView.tsx:6]: done

**Review by virtuman — COMMENTED** [review]
all comments fixed

**Review by danny-avila — APPROVED** [review]

**Comment by virtuman** [comment]
when do you think it will be merged?

**Comment by danny-avila** [comment]
> when do you think it will be merged?

I just merged it, meant to do so earlier. Thank you!

**Pull request merged**